README.adoc

```diff
@@ -1,5 +1,175 @@
-judo-generator-commons
-----------------------
+# Judo Generator Commons
 
 JUDO Generator Commons project contains helpers, annotations and maven utils which can be used on a generator engine development
 
+== Description
```

Comment:
This is already a description. I don't think a separate "description" section is needed